ols/src/utils/token_handler.py

```diff
@@ -49,7 +50,7 @@ def tokens_to_text(self, tokens: list) -> str:
         return self._encoder.decode(tokens)
 
     def truncate_rag_context(
-        self, retrieved_nodes, max_tokens: int = 500
+        self, retrieved_nodes: list[Any], max_tokens: int = 500
```

Comment:
there's no Node type defined?

Reply:
well it is list with this complex values:
```
            {
                "text": "a text text text text",
                "score": 0.6,
                "metadata": {"file_name": "doc1.pdf", "doc_link": "link1"},
            },
```

so it might be `list[dict[str, Any]]`. WDYT?

Reply:
actually no, I'm wrong. It's list of some object. Lemme find it

Reply:
ok so it indeed is/was unknown class. let's start with that one

Reply:
updated